_stbt/core.py

```diff
@@ -1847,6 +1921,10 @@ def _match_template(image, template, match_parameters, roi_mask, level, imglog):
     imglog.add("template", template, level)
     imglog.add("source_matchtemplate", matches_heatmap, level)
 
+    return matches_heatmap
+
+
+def _find_best_match_position(matches_heatmap, method, threshold, level):
```

Comment:
This is quite a bit clearer now that `_match_template` and `_find_best_match_position` are split up.